src/index.js

```diff
@@ -1,106 +1,107 @@
-  log("info", "Successfully logged in");
-  // The BaseKonnector instance expects a Promise as return of the function
-  // log('info', 'Fetching the list of documents')
-  // const $ = await request(`${baseUrl}/index.html`)
-  // // cheerio (https://cheerio.js.org/) uses the same api as jQuery (http://jquery.com/)
-  // log('info', 'Parsing list of documents')
-  // const documents = await parseDocuments($)
-  //
-  // // here we use the saveBills function even if what we fetch are not bills, but this is the most
-  // // common case in connectors
-  // log('info', 'Saving data to Cozy')
-  // await saveBills(documents, fields.folderPath, {
-  //   // this is a bank identifier which will be used to link bills to bank operations. These
-  //   // identifiers should be at least a word found in the title of a bank operation related to this
-  //   // bill. It is not case sensitive.
-  //   identifiers: ['books']
-  // })
+  log('info', 'Authenticating ...')
+  await authenticate(fields.login, fields.password)
+  log('info', 'Successfully logged in')
+  log('info', 'Fetching the list of documents')
+  const $ = await request(`${baseURL.origin}/moncompte/commandes/`)
+  log('info', 'Parsing list of documents')
+  const documents = await parseDocuments($)
+  log('info', 'Saving data to Cozy')
+  await saveBills(documents, fields.folderPath, {
+    identifiers: ['zalando']
+  })
 }
 
-function getCookieValue(cookiesString, cookieName) {
-  const cookieString = RegExp(`${cookieName}[^;]+`).exec(cookiesString)
-  // Return everything after the equal sign, or an empty string if the cookie name not found
-  return decodeURIComponent(!!cookieString ? cookieString.toString().replace(/^[^=]+./,'') : '')
+function parseDocuments($) {
+  const docs = scrape(
+    $,
+    {
+      numero: {
+        sel: 'td.order',
+        parse: content => content.substr(2)
+      },
+      date: {
+        sel: 'td.date',
+        parse: text => moment(text, 'DD/MM/YY').toDate()
+      },
+      amount: {
+        sel: 'td.oTotal',
+        parse: normalizePrice
+      }
+    },
+    '#myOrdersTable tr.cWrapper'
+  )
+  return Promise.all(
+    docs.map(async doc => ({
+      ...doc,
+      filename: createFilename(doc),
+      filestream: await generatePDF(doc),
+      currency: 'EUR',
+      vendor: 'zalando',
+      metadata: {
+        importDate: new Date(),
+        version: 1
+      }
+    }))
+  )
 }
 
 async function getXsrfToken() {
-
-  const loginPage = await request({
+  await request({
     jar,
     method: 'GET',
     url: LOGIN_URL,
     resolveWithFullResponse: true
   })
 
-  const cookiesString = jar.getCookies(LOGIN_URL)
+  const cookiesList = jar.getCookies(LOGIN_URL)
 
-  const xsrfToken = getCookieValue(cookiesString, XSRF_COOKIE_NAME)
-  if (!xsrfToken) throw new Error('XSRF cookie has no value')
+  const xsrfToken = getCookieValue(cookiesList, XSRF_COOKIE_NAME)
+  if (!xsrfToken) {
+    throw new Error('XSRF cookie has no value')
```

Comment:
Idem for error management, log the error and throw errors.VENDOR_DOWN for instance.